addons/l10n_ua/data/account_tax_template.xml

```diff
@@ -0,0 +1,583 @@
+            <field name="description">Дохід за ставкою 15%</field>
+            <field name="amount">0</field>
+            <field name="type_tax_use">sale</field>
+        </record>
+        <!-- Purchase taxes -->
+        <record id="simple_tax_purchase_product_psbo" model="account.tax.template">
+            <field name="chart_template_id" ref="l10n_ua_psbo_chart_template"/>
+            <field name="sequence">40</field>
+            <field name="name">Витрати від продажу  товарів</field>
+            <field name="description">Витрати від продажу товарів</field>
+            <field name="amount">0</field>
+            <field name="type_tax_use">purchase</field>
+        </record>
+        <record id="simple_tax_purchase_salary_psbo" model="account.tax.template">
+            <field name="chart_template_id" ref="l10n_ua_psbo_chart_template"/>
+            <field name="sequence">41</field>
+            <field name="name">Витрати на оплату праці</field>
+            <field name="description">Витрати на оплату праці найманих працівників</field>
+            <field name="amount">0</field>
+            <field name="type_tax_use">purchase</field>
+        </record>
+        <record id="simple_tax_purchase_esv_psbo" model="account.tax.template">
+            <field name="chart_template_id" ref="l10n_ua_psbo_chart_template"/>
+            <field name="sequence">42</field>
+            <field name="name">Витрати ЄСВ</field>
+            <field name="description">Витрати на ЄСВ</field>
+            <field name="amount">0</field>
+            <field name="type_tax_use">purchase</field>
+        </record>
+        <record id="simple_tax_purchase_other_psbo" model="account.tax.template">
+            <field name="chart_template_id" ref="l10n_ua_psbo_chart_template"/>
+            <field name="sequence">43</field>
+            <field name="name">Витрати  інші</field>
+            <field name="description">Витрати інші</field>
+            <field name="amount">0</field>
+            <field name="type_tax_use">purchase</field>
+        </record>
+        <!-- salary taxes -->
+        <!-- esv -->
+        <record id="salary_tax_esv22_psbo" model="account.tax.template">
+            <field name="chart_template_id" ref="l10n_ua_psbo_chart_template"/>
+            <field name="sequence">100</field>
+            <field name="name">ЄСВ  22%</field>
+            <field name="description">ЄСВ 22%</field>
+            <field name="amount">0</field>
+            <field name="type_tax_use">purchase</field>
+            <field name="tag_ids" eval="[(6,0,[ref('l10n_ua.tax_tag_esv')])]"/>
+        </record>
+        <record id="salary_tax_esv_inval_psbo" model="account.tax.template">
+            <field name="chart_template_id" ref="l10n_ua_psbo_chart_template"/>
+            <field name="sequence">101</field>
+            <field name="name">ЄСВ  8,41%</field>
+            <field name="description">ЄСВ для інвалідів</field>
+            <field name="amount">0</field>
```

Comment:
add amount all over the taxes, it's not easy push PR in core let make it  full as much as we can

Reply:
We don't put salary taxes in the tax templates normally.  Mostly, the rules for salaries are more complex.  Most important is VAT.  So, maybe we can just remove those?

Reply:
Yes, This is the salary tax.
It is used only in Salary Rules for accounting moves.
I think it is better to remove all salary taxes as there is no sense to use generic tax report for salary taxes.
We have separate tax report for salary which works without tax_ids.